usaspending_api/etl/management/commands/update_file_c_linkages.py

```diff
@@ -1,22 +1,28 @@
+import logging
+from datetime import datetime
+
 from django.core.management.base import BaseCommand
-from django.db import transaction
+from django.db import transaction, connection
 
-from usaspending_api.common.helpers.etl_helpers import update_c_to_d_linkages
+from usaspending_api.common.helpers.etl_helpers import update_c_to_d_linkages, read_sql_file
+from usaspending_api.common.exceptions import InvalidParameterException
 
 
 class Command(BaseCommand):
 
     LINKAGE_TYPES = ["contract", "assistance"]
     ETL_SQL_FILE_PATH = "usaspending_api/etl/management/sql/"
+    logger = logging.getLogger("script")
 
     def add_arguments(self, parser):
         parser.add_argument(
             "--submission-ids", help=("One or more Broker submission_ids to be updated."), nargs="+", type=int
         )
+        parser.add_argument("--file_d_tablename", help=("Name of the File D table to use."), type=str)
```

Comment:
Can you make this argument first and required?

Reply:
I just pushed a commit to make these changes. I thought I had originally marked this as required, but apparently not.

2847d1b84b1dc515e5eb34a62cfd7bc8cfe2a498